### PR TITLE
Config_tools: Update board xml

### PR DESCRIPTION
**Pull request by Kunhui-Li**
Add nuc11tnbi5.xml;
Update adl-rvp.xml, cfl-k700-i7.xml, ehl-crb-b.xml, nuc7i7dnb.xml, tgl-rvp.xml and whl-ipc-i7.xml.

Tracked-On: #5922
Signed-off-by: Kunhui Li <kunhuix.li@intel.com>
Reviewed-by: Junjie Mao <junjie.mao@intel.com>

### CODE REVIEW

**Comment by wenlingz** [comment]
@Kunhui-Li build fail, please refer:

gdwarf-2 debug//uart16550.c -o /home/meta-acrn/workspace/build/master-acrn-sos/work/intel_corei7_64-oe-linux/acrn-hypervisor/6036-1621487819/build//hypervisor/debug//uart16550.o -MMD -MT /home/meta-acrn/workspace/build/master-acrn-sos/work/intel_corei7_64-oe-linux/acrn-hypervisor/6036-1621487819/build//hypervisor/debug//uart16550.o
| debug//uart16550.c: In function 'uart16550_set_baud_rate':
| debug//uart16550.c:111:32: error: 'uart' undeclared (first use in this function)
|   111 |  temp_reg = uart16550_read_reg(uart, UART16550_LCR);
|       |                                ^~~~
| debug//uart16550.c:111:32: note: each undeclared identifier is reported only once for each function it appears in
| debug//uart16550.c: In function 'uart16550_init':
| debug//uart16550.c:128:7: error: 'uart' undeclared (first use in this function)
|   128 |  if (!uart.enabled) {
|       |       ^~~~
| debug//uart16550.c: In function 'uart16550_getc':
| debug//uart16550.c:194:7: error: 'uart' undeclared (first use in this function)
|   194 |  if (!uart.enabled) {
|       |       ^~~~
| debug//uart16550.c: In function 'uart16550_putc':
| debug//uart16550.c:219:28: error: 'uart' undeclared (first use in this function)
|   219 |   reg = uart16550_read_reg(uart, UART16550_LSR);
|       |                            ^~~~
| debug//uart16550.c: In function 'uart16550_puts':
| debug//uart16550.c:232:7: error: 'uart' undeclared (first use in this function)
|   232 |  if (!uart.enabled) {
|       |       ^~~~
| debug//uart16550.c: In function 'uart16550_set_property':
| debug//uart16550.c:252:2: error: 'uart' undeclared (first use in this function)
|   252 |  uart.enabled = enabled;
|       |  ^~~~
| debug//uart16550.c: In function 'is_pci_dbg_uart':
| debug//uart16550.c:271:6: error: 'uart' undeclared (first use in this function)
|   271 |  if (uart.enabled && (uart.bdf.value != 0)) {
|       |      ^~~~
| debug/Makefile:24: recipe for target '/home/meta-acrn/workspace/build/master-acrn-sos/work/intel_corei7_64-oe-linux/acrn-hypervisor/6036-1621487819/build//hypervisor/debug//uart16550.o' failed
| make[1]: *** [/home/meta-acrn/workspace/build/master-acrn-sos/work/intel_corei7_64-oe-linux/acrn-hypervisor/6036-1621487819/build//hypervisor/debug//uart16550.o] Error 1
| make[1]: Leaving directory '/home/meta-acrn/workspace/build/master-acrn-sos/work/intel_corei7_64-oe-linux/acrn-hypervisor/6036-1621487819/git/hypervisor'
| Makefile:441: recipe for target '/home/meta-acrn/workspace/build/master-acrn-sos/work/intel_corei7_64-oe-linux/acrn-hypervisor/6036-1621487819/build//hypervisor/modules/libdebug.a' failed
| make: *** [/home/meta-acrn/workspace/build/master-acrn-sos/work/intel_corei7_64-oe-linux/acrn-hypervisor/6036-1621487819/build//hypervisor/modules/libdebug.a] Error 2
| make: Leaving directory '/home/meta-acrn/workspace/build/master-acrn-sos/work/intel_corei7_64-oe-linux/acrn-hypervisor/6036-1621487819/git/hypervisor'
| WARNING: /home/meta-acrn/workspace/build/master-acrn-sos/work/intel_corei7_64-oe-linux/acrn-hypervisor/6036-1621487819/temp/run.do_compile.2006:1 exit 1 from 'exit 1'
| 
NOTE: recipe acrn-hypervisor-6036-1621487819: task do_compile: Failed
ERROR: Task (/home/meta-acrn/workspace/meta-acrn/recipes-core/acrn/acrn-hypervisor.bb:do_compile) failed with exit code '1'

**Comment by wenlingz** [comment]
OK to verify

**Comment by wenlingz** [comment]
OK to verify

**Comment by wenlingz** [comment]
@Kunhui-Li please refer, it failed when generate the launch scripts, 

nuc7i7dnb-uefi-industry/log
19:21:13 LOG: cp /home/acrn/workspace/pme_yocto3/build/master-acrn-sos/log/cooker/intel-corei7-64/console-latest.log /home/acrn/workspace/workspace/acrn-hypervisor-premerge-yocto-build-image_3/artifact/6036/yocto-nuc7i7dnb-uefi-industry/log/bitbake-acrn-hypervisor.log
Traceback (most recent call last):
  File "/home/meta-acrn/workspace/build/master-acrn-sos/work/intel_corei7_64-oe-linux/acrn-hypervisor/6036-1621509122/git/misc/config_tools/launch_config/launch_cfg_gen.py", line 246, in <module>
    err_dic = main(ARGS)
  File "/home/meta-acrn/workspace/build/master-acrn-sos/work/intel_corei7_64-oe-linux/acrn-hypervisor/6036-1621509122/git/misc/config_tools/launch_config/launch_cfg_gen.py", line 231, in main
    err_dic = generate_script_file(names, pt_sel, virt_io.dev, dm.args, post_vm_i, config)
  File "/home/meta-acrn/workspace/build/master-acrn-sos/work/intel_corei7_64-oe-linux/acrn-hypervisor/6036-1621509122/git/misc/config_tools/launch_config/launch_cfg_gen.py", line 144, in generate_script_file
    com.gen(names, pt_sel, virt_io, dm, vmid, config)
  File "/home/meta-acrn/workspace/build/master-acrn-sos/work/intel_corei7_64-oe-linux/acrn-hypervisor/6036-1621509122/git/misc/config_tools/launch_config/com.py", line 666, in gen
    pt.gen_pt_head(names, dm, pt_sel, vmid, config)
  File "/home/meta-acrn/workspace/build/master-acrn-sos/work/intel_corei7_64-oe-linux/acrn-hypervisor/6036-1621509122/git/misc/config_tools/launch_config/pt.py", line 242, in gen_pt_head
    if not sel.vpid[pt_dev] or not sel.vpid[pt_dev][vmid]:
KeyError: 2
++ upload /home/acrn/workspace/workspace/acrn-hypervisor-premerge-yocto-build-image_3/artifact/6036 sit-image.sh.intel.com /home/sit/workspace/images/pme-build/acrn-hypervisor sit 123456
++ [[ 5 -ne 5 ]]
++ local src=/home/acrn/workspace/workspace/acrn-hypervisor-premerge

**Comment by yuchuyang** [comment]
@Kunhui-Li  could you please get whl-i5-ipc board in your patch as well? Looks like the this platform is needed for CI.

File: misc/config_tools/data/whl-ipc-i5/whl-ipc-i5.xml [hunk 1]
```diff
@@ -1,62 +1,55 @@
...
 	Region 4: Memory at a1000000 (64-bit, non-prefetchable) [size=1M]
 	00:1f.4 SMBus: Intel Corporation Device 9da3 (rev 30)
-	Region 0: Memory at a1418000 (64-bit, non-prefetchable) [size=256]
+	Region 0: Memory at a1318000 (64-bit, non-prefetchable) [size=256]
 	00:1f.5 Serial bus controller [0c80]: Intel Corporation Device 9da4 (rev 30)
 	Region 0: Memory at fe010000 (32-bit, non-prefetchable) [size=4K]
-	02:00.0 Non-Volatile memory controller: Silicon Motion, Inc. Device 2263 (rev 03)
-	Region 0: Memory at a1300000 (64-bit, non-prefetchable) [size=16K]
-	03:00.0 Ethernet controller: Intel Corporation I210 Gigabit Network Connection (rev 03)
+	01:00.0 Ethernet controller: Intel Corporation I210 Gigabit Network Connection (rev 03)
```
Comment (yuchuyang) [inline misc/config_tools/data/whl-ipc-i5/whl-ipc-i5.xml:45]: @Kunhui-Li 
Does it intentionally remove the nvme (02:00.0 Non-Volatile memory controller) here?
Looks like 
the line33 of https://github.com/projectacrn/acrn-hypervisor/blob/master/misc/config_tools/data/whl-ipc-i5/industry_launch_1uos_hardrt.xml 
the line 75 of https://github.com/projectacrn/acrn-hypervisor/blob/master/misc/config_tools/data/whl-ipc-i5/industry_launch_2uos.xml
The line 75 of https://github.com/projectacrn/acrn-hypervisor/blob/master/misc/config_tools/data/whl-ipc-i5/industry_launch_6uos.xml

DO passthrough this nvme device...

Shall we fix the launch scripts' xml  or shall we fix this board.xml?

**Review by yuchuyang — CHANGES_REQUESTED** [review]

File: misc/config_tools/data/whl-ipc-i5/whl-ipc-i5.xml (hunk 1 above)
Comment (Kunhui-Li) [reply misc/config_tools/data/whl-ipc-i5/whl-ipc-i5.xml]: Updated board.xml for it.

**Comment by wenlingz** [comment]
@terryzouhao please help approve this PR, thanks

**Review by NanlinXie — APPROVED** [review]

**Pull request merged**